Extend/Language.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ *
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * Extends the OXID Language class
+ *
+ * @mixin \OxidEsales\Eshop\Core\Language
+ *
+ * @since 1.0.1
+ */
+class Language extends Language_parent
+{
+    const TRANSLATION_KEY_PREFIX = 'wdpg_';
+    const FALLBACK_LANGUAGE_ABBR = 'en';
+
+    /**
+     * @var int
+     *
+     * @since 1.0.1
+     */
+    private $_iFallbackId = null;
+
+    /**
+     * @inheritdoc
+     *
+     * Provides a fallback language string if the chosen language is not available for the plugin
+     *
+     * @param string $sStringToTranslate Initial string
+     * @param int    $iLang              optional language number
+     * @param bool   $blAdminMode        force to load language constant from admin/shops language file
+     *
+     * @return string
+     *
+     * @since 1.0.1
+     */
+    public function translateString($sStringToTranslate, $iLang = null, $blAdminMode = null)
+    {
+        if (strpos($sStringToTranslate, self::TRANSLATION_KEY_PREFIX) === 0) {
```

Comment:
I would rather do something like this:
```suggestion
        $sParentString = parent::translateString($sStringToTranslate, $iLang, $blAdminMode);
        if (<not translated> && strpos($sStringToTranslate, self::TRANSLATION_KEY_PREFIX) === 0) {
                $sParentString = $this->_getFallbackString($sStringToTranslate);
        }
        
        return  $sParentString;
```